include/config/config_graphics.h

```diff
@@ -30,9 +30,9 @@
 #define IA8_30FPS_COINS
 
 // Use .rej microcode for certain objects (experimental - only should be used when F3DZEX_GBI_2 is defined).
-// So far - recent tests have show that this has reduced performance overall due to CPU load,
-// though in some cases it may perform better.
-// #define OBJECTS_REJ
+// For advanced users only. Does not work perfectly out the box, best used when exported actor models are
+// using 64 vertex sizes, offered by Fast64 in the microcode menu.
+ #define OBJECTS_REJ
```

Comment:
I don't have it in my heart to default this to being on yet

Reply:
That's what we call in the business "an oopsie woopsie"

Reply:
> I don't have it in my heart to default this to being on yet

or ever